automation/taskcluster/decision_task.py

```diff
@@ -218,39 +219,111 @@ def _get_release_gradle_tasks(module_name, is_snapshot):
     return gradle_tasks + ' ' + module_name + ':publish' + ' ' + module_name + ':zipMavenArtifacts'
 
 
-def release(artifacts_info, version, is_snapshot, is_staging):
-    version = components_version() if version is None else version
+# XXX: TO DELETE once bug 1558795 is fixed in early Q3
+def release_snapshot(components, is_snapshot, is_staging):
+    version = components_version()
```

Comment:
I forked this from default `release()` so that we don't have to add too many `if snapshot do this else do that` kind of statements. We're adding snapshot support soon anyway so these two functions will be merged into a single one anyway.